.github/workflows/pitest_two.yml

```diff
@@ -0,0 +1,106 @@
+name: "Pitest2"
```

Comment:
please also change method names to be more meaningful. Also, it may be better to put them all in one file named "Pitest.yml"

Reply:
Done